### PR TITLE
Add tests for etcd-launcher

### LINKED ISSUE #5803: Add automated tests for etcd operations
**User Story**
Currently, new updates to the etcd workflow are tested manually. We need to add automated tests for the new features.
**Acceptance criteria**
The following features should be automatically tested:
- [ ] Etcd cluster resize (up/down)
- [ ] Recover Volume loss.
- [ ] backup and restore

### PR DESCRIPTION
**Pull request by moelsayed**
**What this PR does / why we need it**:
Add tests to ensure the basic functionality behind etcd-launcher.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #5803

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/conformance-tests/etcdlauncher.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+			return false, nil
+		}
+		return active, nil
+	}); err != nil {
+		return fmt.Errorf("failed to check etcd-launcher enabled status: %v", zap.Error(err))
+	}
+	log.Info("etcd-launcher enabled successfully...")
+
+	log.Info("waiting for etcd to regain quorum...")
+	time.Sleep(2 * time.Minute)
```
Comment (lsviben) [inline cmd/conformance-tests/etcdlauncher.go:64]: is there a way to check this with some polling?

**Review by lsviben — COMMENTED** [review]
Looks ok, just left a question

File: cmd/conformance-tests/etcdlauncher.go (hunk 1 above)
Comment (moelsayed) [reply cmd/conformance-tests/etcdlauncher.go:64]: technically, there is, but it's too complicated. It's much simpler to just give it some time.

**Pull request merged**